_data/team.yml

```diff
@@ -160,6 +136,32 @@
 
 # Alumni Section
 
+- name: Nutcha Wattanachit
+  role: PhD Student (2018-2023)
+  description: |
+
+  image: /images/people/nutcha.jpg
+  type: Alumni
+  links: []
+
+- name: Ariane Stark
+  role: MS Student (2020-2023)
+  description: |
+
+  image: /images/people/ariane.jpg
+  type: Alumni
+  links:
+    - name: alumni_link
+      url: "https://www.linkedin.com/in/arianestark/"
+
+- name: Quinn White
+  role: Undergraduate Student (2022-2023)
+  description: |
+
+  image: /images/people/quinn.png
+  type: Alumni
+  links: []
```

Comment:
I think maybe the links should just be left out when there are none? See, e.g. Xi's entry.

Reply:
Hi Nick. I went through all the alumni links, and removed (or changed) those that didn't work, and left only one link per alum(nus), since you had once told me that was the sop (I tried to keep the link that seemed most recently updated or used). I didn't delete the `links: []` lines, because it seems that was the standard formatting for people who didn't have links, from the earliest Reich Lab alumni. And it seems to work (e.g., Harley Jean, Willow Crawford-Crudell). I deleted Xi's link, since it was clearly broken. Let me know if this works for you.